test/e2e/e2ethanos/services.go

```diff
@@ -735,7 +735,6 @@ func NewCompactor(e e2e.Environment, name string, bucketConfig client.BucketConf
 				"--selector.relabel-config": string(relabelConfigBytes),
 				"--wait":                    "",
 			}), extArgs...)...),
-			Readiness:        e2e.NewHTTPReadinessProbe("http", "/-/ready", 200, 200),
```

Comment:
Is this change related to this pr?

Reply:
Nope, was just testing the change with this, slipped into the PR. Reverted.